### ISSUE TITLE
App Crashes when launched without network

### ISSUE THREAD

**Issue by andythomp** [vijaysharm/resume-android#1]
The app crashes when launched without network. 

Nice UI though! Very slick.

```
E/AndroidRuntime(24486): FATAL EXCEPTION: main
E/AndroidRuntime(24486): Process: ca.vijaysharma.resume, PID: 4911
E/AndroidRuntime(24486): java.lang.RuntimeException: Unable to start activity ComponentInfo{ca.vijaysharma.resume/ca.vijaysharma.resume.ResumeActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object java.util.Map.get(java.lang.Object)' on a null object reference
E/AndroidRuntime(24486):    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2325)
E/AndroidRuntime(24486):    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2390)
E/AndroidRuntime(24486):    at android.app.ActivityThread.access$800(ActivityThread.java:151)
E/AndroidRuntime(24486):    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
E/AndroidRuntime(24486):    at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(24486):    at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime(24486):    at android.app.ActivityThread.main(ActivityThread.java:5257)
E/AndroidRuntime(24486):    at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(24486):    at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime(24486):    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
E/AndroidRuntime(24486):    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
E/AndroidRuntime(24486): Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object java.util.Map.get(java.lang.Object)' on a null object reference
E/AndroidRuntime(24486):    at ca.vijaysharma.resume.ResumeData.v(ResumeData.java:421)
E/AndroidRuntime(24486):    at ca.vijaysharma.resume.ResumeData.v(ResumeData.java:417)
E/AndroidRuntime(24486):    at ca.vijaysharma.resume.ResumeData.projects(ResumeData.java:297)
E/AndroidRuntime(24486):    at ca.vijaysharma.resume.ResumeActivity.onCreate(ResumeActivity.java:59)
E/AndroidRuntime(24486):    at android.app.Activity.performCreate(Activity.java:5990)
E/AndroidRuntime(24486):    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1106)
E/AndroidRuntime(24486):    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2278)
E/AndroidRuntime(24486):    ... 10 more
```

Here's a log

**Comment by vijaysharm** [comment]
Thanks @andythomp! Maybe i can package a version with my app so, even without an internet connection to get the resume.json, I can still show something (even if its not up-to-date).

**Comment by vijaysharm** [comment]
Fixed! Under changeset e19dcde.
